src/gs-details-page.c

```diff
@@ -2079,6 +2210,71 @@ gs_details_page_write_review_cb (GtkButton *button,
 	gs_shell_modal_dialog_present (self->shell, GTK_DIALOG (dialog));
 }
 
+static gconstpointer
+find_mount_in_list (GList  *mounts,
+                    GMount *needle)
+{
+	GList *l;
+	g_autofree gchar* uuid_needle = g_mount_get_uuid (needle);
+
+	for (l = mounts; l; l = l->next) {
+		g_autofree gchar *uuid_cur = g_mount_get_uuid (l->data);
+
+		if (g_strcmp0 (uuid_needle, uuid_cur) == 0) {
+			return l->data;
+		}
+	}
+
+	return NULL;
+}
+
+static void
+gs_details_page_mount_added_cb (GVolumeMonitor *volume_monitor,
+				GMount *mount,
+				GsDetailsPage *self)
+{
+	g_autoptr(GDrive) drive = g_mount_get_drive (mount);
+	g_autoptr(GFile) root = g_mount_get_root (mount);
+	g_autofree gchar *path = g_file_get_path (root);
+	g_autoptr(GMount) mount_existing = NULL;
```

Comment:
I usually name variables `noun_adjective` since the noun is usually more important and it makes similar variables easier to visually group (eg, if we were re-mounting a device in a new location and had "mount_existing" and "mount_new").

But it's also not a big deal to me and it matters less when there's a single variable.

Fixed locally.